### PR TITLE
Updating wiki -> Latex 

### PR DESCRIPTION
**Pull request by albeanth**
@rtrp/admins 

I was making a new cleaner latex style file for myself and decided to put it up on the wiki and see what you guys think. Maybe add your own stuff in and we can build something kind of cool. 

Anyways, I moved some other things around too while I was in there:
  - I reorganized the Latex section of the wiki. Added some overleaf/sharelatex links, and created an examples section. 
  - That section has a general style file for general document creation and a thesis example. 
  - @dougwoods3 I moved the .zip files you had into a new directory in the wiki. Hope that didn't screw with things for you. 

If you guys want to reformat things feel free. I just did this real fast to get a conversation going. Also, feel free to add things into the style file. Basically the idea is for users to open a tex document, and have the following:

\documentclass{article} % or whatever
\usepackage{standard}

% include "non-standard" packages that are document specific

\begin{document}
Stuff and things.
\end{document}

### CODE REVIEW

File: wiki/LaTeX_docs/standard.sty [hunk 1]
```diff
@@ -0,0 +1,23 @@
+\usepackage[utf8]{inputenc}
```
Comment (dougwoods3) [inline wiki/LaTeX_docs/standard.sty:1]: Can you add a comment to each of these other ones to describe what they're for?

File: wiki/index.md [hunk 2]
```diff
@@ -39,12 +39,17 @@ A brief and non-exhaustive list of software NSE has access to. Please note that
...
-  - A tool to create technical presentations that is based on LaTeX.
-  - If you want to get into beamer, feel free to check out this [reference](./references/beameruserguide.pdf) and the [example](../users/woodsdou/DougsDefense.zip) that Doug Woods has provided.
+  - When you are feeling collaborative (or are forced to for course work) and still want to use LaTeX, [ShareLaTeX](https://www.sharelatex.com) and [Overleaf](https://www.overleaf.com/) are great resources. This year, [the two joined forces](https://www.sharelatex.com/blog/2017/07/20/sharelatex-joins-overleaf.html) so eventually they may(?) be one in the same.
+
+    #### LaTeX Examples
+    * Basic style file:
+      - Here is a [basic style file](./LaTeX_docs/standard.sty) that is used to format a LaTeX file.
+      - Doug Woods has been kind enough to supply a [LaTeX Demo](./LaTeX_docs/LatexDemo.zip) for new users to learn tex.
+    * MS Thesis
+      - You can use his [MS Thesis in LaTeX](./LaTeX_docs/DougsMSThesis.zip) as a template. Execute pdflatex on MS_Thesis.tex to generate the pdf.
```
Comment (dougwoods3) [inline wiki/index.md:49]: Actually, you probably have to execute:
pdflatex MS_Thesis.tex
bibtex MS_Thesis.tex
pdflatex MS_Thesis.tex
pdflatex MS_Thesis.tex

**Comment by dougwoods3** [comment]
I like the reformatting and reorganization. I say :shipit: after Jackson has a peek.

**Comment by harterj** [comment]
Yeah, let me take a look tomorrow after I'm back from my fishing trip. I scanned it a bit and I like the changes so far 👍

**Comment by albeanth** [comment]
@dougwoods3 - Looks good what you added in. I like the latter commit with the organization. 

@harterj - look things over and then once you are happy with it go ahead and 🚢  it.

**Comment by harterj** [comment]
@albeanth @dougwoods3 looks good to me! Nice that you spent the time to clean it up and update it. :shipit:

**Pull request merged**